src/olympia/devhub/templates/devhub/new-landing/components/footer.html

```diff
@@ -37,7 +37,7 @@ <h3 class="DevHub-Footer-sections-header">{{ _('Resources') }}</h3>
         <h4>{{ _('Tools') }}</h4>
         <ul>
           <li><a href="{{ url('devhub.validate_addon') }}">{{ _('Add-on Validator') }}</a></li>
-          <li><a href="https://addons.mozilla.org/developers/addon/check-compatibility">{{ _('Compatibility Checker') }}</a></li>
+          <li><a href="addon/check-compatibility">{{ _('Compatibility Checker') }}</a></li>
```

Comment:
I think this should rather be `{% url('devhub.check_addon_compatibility') %}`, there was a reason why we didn't use that in the past but I don't remember. As long as @atsay is on board (and I think this was requested by QA?)